landoapi/models/treestatus.py

```diff
@@ -150,31 +143,14 @@ class Log(Base):
     changed_by = db.Column(db.Text, nullable=False)
 
     # The status which the tree has been set to.
-    status = db.Column(
-        db.Enum(
-            TreeStatus,
-            # Use the values of the enum in the DB instead of the keys.
-            values_callable=lambda x: [i.value for i in x],
-        ),
-        nullable=False,
-    )
+    status = db.Column(db.Enum(TreeStatus), nullable=False)
```

Comment:
Does this need a default?

Reply:
No, there shouldn't be a default here - `status` isn't set that should be considered a programming error.